app/concerns/orderable.rb

```diff
@@ -36,6 +36,7 @@ def reorder_others(is_after)
       others.each_with_index do |other, index|
         next if other.order == index
         other.order = index
+        other.order += 1 if other.is_a?(Reply) && post.written.blank?
```

Comment:
Could this use `post.written.nil?` instead? Or is this some kind of builder association? (To confirm my understanding)

Reply:
Looks like it's a builder association!